samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -80,6 +95,29 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
         });
     };
 
+    const selectDocument = () => {
+        documentFileRef.current?.click();
+    };
+
+    const importDocument = async () => {
+        const documentFile = documentFileRef.current?.files?.[0];
+        if (documentFile) {
+            try {
+                SetDocumentImporting(true);
+                await documentImportService.importDocumentAsync(
+                    documentFile,
+                    await AuthHelper.getSKaaSAccessToken(instance)
+                );
+                dispatch(addAlert({ message: 'Document uploaded successfully', type: AlertType.Success }));
+                // TODO: Add document import message to the chat
```

Comment:
The notification at the top of the chat is sufficient - I think you can remove this TODO.